could not find `windows` in `os`

My code on Windows:
```rust
use muddy::{m, muddy_init};
muddy_init!();

fn main() {
    println!("{}", m!("My highly obfuscated text"));
    println!("{}", "My non obfuscated static str - ripgrep me");
}
```
Run `cargo build --target x86_64-unknown-linux-musl`
```
error[E0433]: failed to resolve: could not find `windows` in `os`                                                                                              
   --> src\main.rs:2:1
    |
2   | muddy_init!();
    | ^^^^^^^^^^^^^ could not find `windows` in `os`
    |
note: found an item that was configured out
   --> C:\Users\myname\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library/std/src/os/mod.rs:53:9
    |
53  | pub mod windows {}
    |         ^^^^^^^
note: found an item that was configured out
   --> C:\Users\myname\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library/std/src/os/mod.rs:100:9
    |
100 | pub mod windows;
    |         ^^^^^^^
    = note: this error originates in the macro `muddy_init` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0433`.                                                                                            
error: could not compile `test_muddy` (bin "test_muddy") due to 1 previous error    
```

Expand macros:
```rust
pub mod muddy_internal {
    use muddy::{GenericArray, KeyInit, Lazy, ChaCha20Poly1305, Key, Aead, U32, Nonce};
    pub use muddy::{LazyStr, FromHex};
    use std::os::windows::ffi::OsStrExt;

    static KHJU9SXTVNMVXTDQY: &'static [u8; 32] = b"s|i\xbc\xa88\x9et\x18[}\xb1\xbdn\xdfP\xce\xe0\rH\xf47\x9am,{\xa6\xe4\xe2=\x9c\x15";
    static JUNK: &'static [u8; 32] = b"\xff\x89 \x9d!\xb2\x0ef\xb0\x87t)\xf6\x8e\x01\xc8\xee^\xca2:Z\x1a\x1a\x92\xc0\xc7\xc4(\x03I ";
    static PMBKW4TKD2HJK04P8: Lazy<ChaCha20Poly1305> = Lazy::new(|| {
        let mut key = KHJU9SXTVNMVXTDQY.clone();
        key.iter_mut().zip(JUNK.iter()).for_each(|(b, k)| *b ^= k);
        ChaCha20Poly1305::new(Key::from_slice(&key))
    });

    pub fn decrypt(encrypted: &[u8], nonce: &[u8]) -> String {
        let nonce = Nonce::from_slice(nonce);
        let plaintext = PMBKW4TKD2HJK04P8.decrypt(nonce, encrypted).unwrap();
        String::from_utf8(plaintext).unwrap()
    }
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+* support multiple target families (#20)
 
 ## 0.2.1
 * add `pull_request.yml` to check for added `CHANGELOG.md` entries on pull requests (#8)

--- a/muddy_macros/src/internal.rs
+++ b/muddy_macros/src/internal.rs
@@ -72,22 +72,15 @@ pub fn build_obfuscation_mod(
 
 /// Creates the inports for the `muddy_internal` mod
 fn build_obfuscation_imports() -> proc_macro2::TokenStream {
-    #[cfg(unix)]
-    let use_os_str_ext = quote! {
-        use std::os::unix::ffi::OsStrExt;
-    };
-    #[cfg(target_os = "wasi")]
-    let use_os_str_ext = quote! {
-        use std::os::wasi::ffi::OsStrExt;
-    };
-    #[cfg(windows)]
-    let use_os_str_ext = quote! {
-        use std::os::windows::ffi::OsStrExt;
-    };
     quote! {
         use muddy::{GenericArray, KeyInit, Lazy, ChaCha20Poly1305, Key, Aead, U32, Nonce};
         pub use muddy::{LazyStr, FromHex};
-        #use_os_str_ext
+        #[cfg(target_family = "unix")]
+        use std::os::unix::ffi::OsStrExt;
+        #[cfg(target_family = "windows")]
+        use std::os::windows::ffi::OsStrExt;
+        #[cfg(target_family = "wasm")]
+        use std::os::wasi::ffi::OsStrExt;
     }
 }
 

